kqueen/config/demo.py

```diff
@@ -4,6 +4,8 @@
 class Config(BaseConfig):
     DEBUG = False
     LOG_LEVEL = 'INFO'
+    LOG_PATH = 'log/kqueen'
```

Comment:
Please move logs out of source code.

Reply:
why? if i remove it from all configs, api should not start. If we decided to provide logging in files, we should provide default path's , otherwise we should provide some kind of additional checker 'FILE_LOGGING_ON/OFF' and wrap all stuff into it

Reply:
I just wanted to log into different directory, something like "/var/log/queen/ or anything else.

Reply:
oh, got it